src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/GlissandoExpandableBlank.qml

```diff
@@ -35,6 +35,8 @@ ExpandableBlank {
 
     enabled: model ? !model.isEmpty : false
 
+    property bool isHarpGliss: model ? model.isHarpGliss : false
```

Comment:
I'd make this a `readonly` property and put it above `enabled` to keep it together with other properties